api/src/main/java/io/opencensus/trace/Tracing.java

```diff
@@ -89,6 +90,19 @@ public static TraceConfig getTraceConfig() {
   // Any provider that may be used for TraceComponent can be added here.
   @DefaultVisibilityForTesting
   static TraceComponent loadTraceComponent(@Nullable ClassLoader classLoader) {
+    try {
+      // Call Class.forName with literal string name of the class to help shading tools.
+      return Provider.createInstance(
+          Class.forName(
+              "migration.OpenTelemetryTraceComponentImpl", /*initialize=*/ true, classLoader),
```

Comment:
Supposedly, the actual package will be different? Something like `com.google.cloud.opentelemetry.trace.opencensus.OpenTelemetryTraceComponent` (as part of https://github.com/GoogleCloudPlatform/opentelemetry-operations-java)?

Reply:
yep I'll change this once I put up the migration library PR :)

Reply:
Changed this here as @james-bebbington suggested moving the migration library to https://github.com/open-telemetry/opentelemetry-java

Reply:
Did you change it though?

Reply:
Do you mean the class name here, or the migration library in `opentelemetry-java`? I've changed the class name here to `io.opentelemetry.opencensusshim.OpenTelemetryTraceComponentImpl`, I have the changes to `opentelemetry-java` locally, will make a PR when they're ready, but for now it should be okay to set it here first right?

Reply:
Ah yep, sorry.
I meant the package name, and this file is outdated now, so I've been looking in the wrong place.
Apologies for confusion.